frontend/src/pages/spotnet/dashboard/Dashboard.jsx

```diff
@@ -120,7 +120,7 @@ const [isCollateralActive, setIsCollateralActive] = useState(true);
       <div className="dashboard-container">
         {loading && <Spinner loading={loading} />}
         <h1 className="dashboard-title">zkLend Position</h1>
-        <div className="main-container">
+        <div className="dashboard-content">
           <div className="top-cards">
             <div className="card">
```

Comment:
2 cards should fit scren on 390px screen, you can see that currently their size is larger. Please check and implement according to figma

Reply:
Also currently cards has more padding the block below, but they should have the same width and padding (see Figma)